build/build-uniqush-push.sh

```diff
@@ -20,16 +20,20 @@ VERSION=`"$TEMP/bin/uniqush-push" --version | sed 's/uniqush-push //'`
 BUILD=`pwd`/"uniqush-push-$VERSION"
 mkdir -p "$BUILD/usr/bin"
 mkdir -p "$BUILD/etc/uniqush/"
+mkdir -p "$BUILD/systemd/"
+mkdir -p "$BUILD/systemvinit/"
 
 ARCH="`uname -m`"
 
 cp "$TEMP/bin/uniqush-push" "$BUILD/usr/bin"
 cp "$TEMP/src/github.com/uniqush/uniqush-push/conf/uniqush-push.conf" "$BUILD/etc/uniqush"
+cp "../conf/systemd/uniqush-push.service" "$BUILD/systemd/uniqush-push.service"
+cp "../conf/systemvinit/uniqush.init" "$BUILD/systemvinit/uniqush-push"
 cp "$TEMP/src/github.com/uniqush/uniqush-push/LICENSE" "$LICENSE"
 
 fpm -s dir -t rpm -v "$VERSION" -n uniqush-push --license="$LICENSE" --maintainer="Nan Deng" --vendor "uniqush" --url="http://uniqush.org" --category Network --description "Uniqush is a free and open source software which provides a unified push service for server-side notification to apps on mobile devices" -a "$ARCH" -C "$BUILD" .
```

Comment:
does this need an `--exclude="system*"` too?

Reply:
it works either on Systemd and on the old systemV